### ISSUE TITLE
Project Files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [zombiezen/crab-battle#4]
```
Currently, we have no project files, but we will be adding Dev-C++ project 
files.

In addition, it would be nice to be able to compile the project on multiple 
platforms easily.  Ross, 
having access to both a Mac and a Linux machine, will deal with adding project 
files (and possibly 
Makefiles) to the project.

```

Original issue reported on code.google.com by `TJ.Ferr...@gmail.com` on 22 Feb 2008 at 5:30

**Comment by GoogleCodeExporter** [comment]
```
I've added an Xcode project.  We still need a Dev-C++ project file.
```

Original comment by `TJ.Ferr...@gmail.com` on 22 Feb 2008 at 6:19
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I have added a Dev-C++ project file.  Someone should confirm if it works with 
wxDev-C++.

We still need a Makefile, but I'm working on that.
```

Original comment by `rlight2@gmail.com` on 23 Feb 2008 at 4:20

**Comment by GoogleCodeExporter** [comment]
```
As it stands now, the project file does work with wxDev-C++ (I've now switched 
over
to wxDev-C++).  It seems we are still experiencing some issues with getting 
SDL_image
includes correct, but that can wait.

We're not going to have the Linux Makefile by Milestone 1 (I'm not going to be 
able
to do it), but expect a push for it next week, as I will be receiving a Linux 
laptop
(my new primary development machine :)).

Bottom line: Linux Makefile pushed to Milestone 2, everything else seems to 
work.
```

Original comment by `rlight2@gmail.com` on 6 Mar 2008 at 12:42
- Added labels: **Milestone-2**
- Removed labels: **Milestone-1**

**Comment by GoogleCodeExporter** [comment]
```
Just updating this, we have a dev cpp file that works and a linux make file is 
soon
coming
```

Original comment by `TJ.Ferr...@gmail.com` on 20 Mar 2008 at 4:33

**Comment by GoogleCodeExporter** [comment]
```
I finally got my laptop (hooray) and I've added a Linux Makefile (r120).
```

Original comment by `rlight2@gmail.com` on 21 Mar 2008 at 1:26
- Changed state: **Fixed**
